phobos/io/parser.py

```diff
@@ -22,7 +24,7 @@ def parse_xml(xml):
         try:
             xml_root = ET.fromstring(xml_string)
         except ET.ParseError as e:
-            print("ERROR: Tried to parse:\n", xml)
+            log.error(f"ERROR: Tried to parse:\n  {xml}")
```

Comment:
When we log to error we can remove the ERROR part from the string. For the other types of course, too